CIP-1694/README.md

```diff
@@ -0,0 +1,664 @@
+---
+CIP: 1694
+Title: An On-Chain Decentralized Governance Mechanism for Voltaire
+Status: Proposed
+Category: Ledger
+Authors:
+    - Jared Corduan <jared.corduan@iohk.io>
+    - Matthias Benkort <matthias.benkort@cardanofoundation.org>
+    - Kevin Hammond <kevin.hammond@iohk.io>
+    - Charles Hoskinson <charles.hoskinson@iohk.io>
+    - Samuel Leathers<samuel.leathers@iohk.io>
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: 2022-11-18
+License: CC-BY-4.0
+---
+
+# An On-Chain Decentralized Governance Mechanism for Voltaire
+
+## Abstract
+
+We propose a revision of Cardano's on-chain governance system to support the new requirements for Voltaire. The existing specialized governance support for protocol parameter updates and MIR certificates will be deprecated, and two new fields will be added to normal transaction bodies:
+
+1. governance actions;
+2. votes.
+
+**Any Cardano user** will be allowed to submit a **governance action**. Three distinct groups will be responsible for ratifying these governance actions using their **votes**:
+
+1. a Constitutional committee;
+2. a group of delegation representatives (henceforth called **DReps**); and
```

Comment:
I worry that "DRep" might be too close to "derp". Here are some alternatives:

- DCR: Decentralized Cardano Representatives
- DSR: Decentralized Stakeholder Representatives
- DAR: Decentralized Autonomous Representatives
- DGR: Decentralized Governance Representatives
- DLR: Decentralized Ledger Representatives

Reply:
> I worry that "DRep" might be too close to "derp".

That's literally the only thing I like about dreps ;)

Reply:
Could also be Voting Agents (VAs)

Reply:
Some more ideas:
 - vRep (Voltaire Representative)
 - gRep (Governance Representitive)

Or maybe just:
 - Voter

;)

Reply:
@JaredCorduan in explaining this to outsiders I've been using the term _proxy_ because of its usage in corporate voting: https://en.wikipedia.org/wiki/Proxy_voting#Delegated_voting

Reply:
I'll ask @CharlesHoskinson to name them, I usually try to stay out of naming arguments :laughing: